### ISSUE TITLE
New Order Email

### ISSUE THREAD

**Issue by bigfatfrog** [woocommerce/woocommerce#24803]
Changes to the 'New Order' email don't get saved.

![image](https://user-images.githubusercontent.com/6823994/66576770-2ddf8a80-eb70-11e9-9a1f-c6005c7f0a24.png)

Other email maintence forms - cancelled order, failed order ect are working OK

**WordPress Environment**
<details>
```
`
### WordPress Environment ###

WordPress address (URL): https://www.mgf.ltd.uk
Site address (URL): https://www.mgf.ltd.uk
WC Version: 3.7.1
REST API Version: ✔ 1.0.2
Log Directory Writable: ✔
WP Version: 5.2.3
WP Multisite: –
WP Memory Limit: 768 MB
WP Debug Mode: –
WP Cron: ✔
Language: en_US
External object cache: –

### Server Environment ###

Server Info: Apache
PHP Version: 7.1.30 - We recommend using PHP version 7.2 or above for greater performance and security. How to update your PHP version
PHP Post Max Size: 128 MB
PHP Time Limit: 120
PHP Max Input Vars: 3000
cURL Version: 7.59.0
OpenSSL/1.0.2r

SUHOSIN Installed: –
MySQL Version: 5.6.40-84.0-log
Max Upload Size: 257 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 3.7.1
WC Database Prefix: wp_
MaxMind GeoIP Database: ✔
Total Database Size: 80.62MB
Database Data Size: 64.92MB
Database Index Size: 15.70MB
wp_woocommerce_sessions: Data: 11.60MB + Index: 0.41MB + Engine MyISAM
wp_woocommerce_api_keys: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_woocommerce_attribute_taxonomies: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_woocommerce_downloadable_product_permissions: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_woocommerce_order_items: Data: 0.01MB + Index: 0.01MB + Engine MyISAM
wp_woocommerce_order_itemmeta: Data: 0.08MB + Index: 0.06MB + Engine MyISAM
wp_woocommerce_tax_rates: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
wp_woocommerce_tax_rate_locations: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_woocommerce_shipping_zones: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_woocommerce_shipping_zone_locations: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_woocommerce_shipping_zone_methods: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_woocommerce_payment_tokens: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_woocommerce_payment_tokenmeta: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_woocommerce_log: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
midesign: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_blc_filters: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_blc_instances: Data: 0.25MB + Index: 0.05MB + Engine MyISAM
wp_blc_links: Data: 1.33MB + Index: 0.19MB + Engine MyISAM
wp_blc_synch: Data: 0.01MB + Index: 0.01MB + Engine MyISAM
wp_bwps_lockouts: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_bwps_log: Data: 0.49MB + Index: 0.06MB + Engine MyISAM
wp_commentmeta: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_comments: Data: 0.02MB + Index: 0.02MB + Engine MyISAM
wp_doptg_galleries: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_doptg_images: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_doptg_settings: Data: 0.01MB + Index: 0.00MB + Engine MyISAM
wp_failed_jobs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_links: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_mailchimp_carts: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mgmlp_folders: Data: 0.02MB + Index: 0.02MB + Engine MyISAM
wp_options: Data: 1.68MB + Index: 0.18MB + Engine MyISAM
wp_postmeta: Data: 12.69MB + Index: 3.92MB + Engine MyISAM
wp_posts: Data: 17.67MB + Index: 1.14MB + Engine MyISAM
wp_queue: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_rank_math_404_logs: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_rank_math_internal_links: Data: 0.14MB + Index: 0.06MB + Engine InnoDB
wp_rank_math_internal_meta: Data: 0.05MB + Index: 0.00MB + Engine InnoDB
wp_rank_math_redirections: Data: 0.08MB + Index: 0.02MB + Engine InnoDB
wp_rank_math_redirections_cache: Data: 0.06MB + Index: 0.02MB + Engine InnoDB
wp_rank_math_sc_analytics: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_redirection_404: Data: 6.80MB + Index: 2.25MB + Engine MyISAM
wp_redirection_groups: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_redirection_items: Data: 0.03MB + Index: 0.05MB + Engine MyISAM
wp_redirection_logs: Data: 2.66MB + Index: 0.76MB + Engine MyISAM
wp_revslider_static_slides: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_smooth_slider: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_smooth_slider_meta: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_smooth_slider_postmeta: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_termmeta: Data: 0.03MB + Index: 0.03MB + Engine MyISAM
wp_terms: Data: 0.01MB + Index: 0.01MB + Engine MyISAM
wp_term_relationships: Data: 0.11MB + Index: 0.23MB + Engine MyISAM
wp_term_taxonomy: Data: 0.01MB + Index: 0.01MB + Engine MyISAM
wp_usermeta: Data: 0.11MB + Index: 0.06MB + Engine MyISAM
wp_users: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
wp_wc_admin_notes: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_admin_note_actions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_customer_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_download_log: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_product_lookup: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wc_order_stats: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_product_meta_lookup: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_webhooks: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_wpanything_content: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_wpanything_settings: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_wplnst_scans: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
wp_wplnst_scans_objects: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wplnst_urls: Data: 0.44MB + Index: 1.02MB + Engine InnoDB
wp_wplnst_urls_locations: Data: 2.52MB + Index: 2.38MB + Engine InnoDB
wp_wplnst_urls_locations_att: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wplnst_urls_status: Data: 5.47MB + Index: 2.08MB + Engine InnoDB
wp_yoast_seo_links: Data: 0.13MB + Index: 0.04MB + Engine MyISAM
wp_yoast_seo_meta: Data: 0.03MB + Index: 0.03MB + Engine MyISAM

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (37) ###

3CX Live Chat and Talk: by 3CX – 1.4.3
AddToAny Share Buttons: by AddToAny – 1.7.37
Adminimize: by Frank Bültge – 1.11.5
Advanced Custom Fields: by Elliot Condon – 5.8.3
Contact Form 7: by Takayuki Miyoshi – 5.1.4
Thumbnail Gallery (WordPress Plugin): by Dot on Paper – 2.4
Enable Media Replace: by ShortPixel – 3.3.6
EU Cookie Law: by Alex Moss
Marco Milesi
Peadig
Shane Jones – 3.0.6

Export Database: by LittleBizzy – 1.0.9
Flamingo: by Takayuki Miyoshi – 2.1
Insert Headers and Footers: by WPBeginner – 1.4.4
Justified Image Grid (Share on Theme123.Net): by Firsh – 2.2
Log Viewer: by Markus Fischbacher – 14.05.04
Mailchimp for WooCommerce: by Mailchimp – 2.2.4
MC4WP: Mailchimp for WordPress: by ibericode – 4.5.5
Multiple Post Thumbnails: by Chris Scott – 1.7
Nexus Theme Changes: by  –
Revert: by John Blackbourn – 1.0
SEMrush SEO Writing Assistant: by SEMrush – 1.0.4
Rank Math SEO: by Rank Math – 1.0.33
SEO Friendly Images: by Vladimir Prelovac – 3.0.5
SG Optimizer: by SiteGround – 5.3.2
Simple Testimonials: by PIXELovely – 1.0.0
Smooth Slider: by SliderVilla – 2.8.8
Sucuri Security - Auditing, Malware Scanner and Hardening: by Sucuri Inc. – 1.8.21
UpdraftPlus - Backup/Restore: by UpdraftPlus.Com
DavidAnderson – 1.16.17

Increase Maximum Upload File Size: by WebFactory Ltd – 1.35
User Role Editor: by Vladimir Garagulya – 4.51.3
WooCommerce Admin: by WooCommerce – 0.20.0
WooCommerce WorldPay Gateway: by Andrew Benbow – 4.1.0 – 4.1.1 is available – Not tested with the active version of WooCommerce
WooCommerce: by Automattic – 3.7.1
WordPress Importer: by wordpressdotorg – 0.6.4
WP Add Mime Types: by Kimiya Kitani – 2.4.0
wp-jquery-lightbox: by Ulf Benjaminsson – 1.4.8.1
WP Logo Showcase Responsive Slider: by WP OnlineSupport – 2.3.1
WP-Protect: by Little Bizzy – 1.4
YITH WooCommerce Request A Quote Premium: by YITH – 2.2.0

### Inactive Plugins (11) ###

Broken Link Checker: by ManageWP – 1.11.8
Capability Manager Enhanced: by PublishPress – 1.7.5
Export All URLs: by Atlas Gondal – 3.6
Fix Media Library: by WowPress.host – 2.0
Google Analytics: by ShareThis – 2.2.5
Post Template Select: by munishsandy – 1.3
Redirection: by John Godley – 4.4.2
Simple 301 Redirects: by Scott Nellé – 1.07
Unbounce Landing Pages: by Unbounce – 1.0.44
WP Post Type Template: by CyberCraft – 1.0.3
Yoast SEO: by Team Yoast – 12.2

### Dropin Plugins (1) ###

maintenance.php: maintenance.php

### Must Use Plugins (1) ###

nexus-wp-login-screen.php: by  –

### Settings ###

API Enabled: –
Force SSL: ✔
Currency: GBP (£)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: ✔

### WC Pages ###

Shop base: #7897 - /products2/
Cart: #7898 - /cart/
Checkout: #7899 - /checkout/
My account: #7900 - /my-account/
Terms and conditions: #8501 - /website-terms-conditions/

### Theme ###

Name: 
Version: – 0 is available
Author URL: 
Child Theme: ✔
Parent Theme Name: 
Parent Theme Version: – 0 is available
Parent Theme Author URL: 
WooCommerce Support: ✔

### Templates ###

Overrides: MGF/woocommerce/archive-product.php
MGF/woocommerce/single-product.php


### Action Scheduler ###

Complete: 12
Oldest: 2019-06-02 09:50:18 +0000
Newest: 2019-09-17 08:24:57 +0000

Pending: 4
Oldest: 2019-09-24 09:28:43 +0000
Newest: 2019-10-10 13:29:21 +0000

Canceled: 0
Oldest: –
Newest: –

In-progress: 0
Oldest: –
Newest: –

Failed: 0
Oldest: –
Newest: –

`
```
</details>

**Comment by Braehler** [comment]
Just remove the emojy in the additional content tab

**Comment by juliaamosova** [comment]
Hi @bigfatfrog, thanks for reporting the issue! It is a duplicate of https://github.com/woocommerce/woocommerce/issues/24405. The issue had been resolved on our end. [The fix will be included in 3.8](https://github.com/woocommerce/woocommerce/blob/master/readme.txt#L257) which is scheduled to be released on November 5th. For now, please remove the emoji as mentioned by @Braehler.
